### PR TITLE
Added tools for telemetry

### PR DESCRIPTION
**Pull request by semeniak97mf**
Added tools for telemetry: Polkadot telemetry (https://github.com/paritytech/substrate-telemetry), prometheus, grafana.

### CODE REVIEW

File: .github/workflows/deploy-telemetry-devnet.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+
+    - name: Install system deps
+      run: sudo apt-get install gettext
+
+    - name: Set up SSH
```
Comment (hrls) [inline .github/workflows/deploy-telemetry-devnet.yml:18]: We should consider about making these steps as re-usable or dedicated action.

File: .github/workflows/deploy-telemetry-devnet.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        mkdir -p ~/.ssh/
+        touch ~/.ssh/known_hosts
+        ssh-keyscan ${{ secrets.TELEMETRY_HOST }} >> ~/.ssh/known_hosts
+        echo "${{ secrets.TELEMETRY_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+        chmod 600 ~/.ssh/id_rsa
+        eval $(ssh-agent)
+
+    - name: Substitute environment variables
+      run: |
+        export TELEMETRY_HOST=${{ secrets.TELEMETRY_HOST }}
```
Comment (hrls) [inline .github/workflows/deploy-telemetry-devnet.yml:29]: Use `env: ` section instead.

File: .github/workflows/deploy-telemetry-devnet.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      uses: appleboy/ssh-action@v1.0.3
+      with:
+        host: ${{ secrets.TELEMETRY_HOST }}
+        username: ${{ secrets.TELEMETRY_USERNAME }}
+        key: ${{ secrets.TELEMETRY_PRIVATE_KEY }}
+        script: |
+          cd /sportchain
+          docker compose down
+          docker compose pull
+          docker compose up -d --build --remove-orphans
```
Comment (hrls) [inline .github/workflows/deploy-telemetry-devnet.yml:49]: We all know what this `-d` means, but IMO source files should contain full keys like `--detach`.

File: .github/workflows/deploy-telemetry-devnet.yml (hunk 2 above)
Comment (semeniak97mf) [reply .github/workflows/deploy-telemetry-devnet.yml]: Done

File: .github/workflows/deploy-devnet.yml [hunk 4]
```diff
@@ -30,9 +30,14 @@ jobs:
         echo "${LATEST_RELEASE}"
         gh release download "${LATEST_RELEASE}" --repo Atleta-network/atleta --pattern "chain-spec.devnet.json" --dir chainspecs
 
+    - name: Substitute environment variables
+      run: |
+        export DEVNET_HOST=${{ secrets.DEVNET_HOST }}
```
Comment (hrls) [inline .github/workflows/deploy-devnet.yml:35]: TODO `env:`

**Review by hrls — DISMISSED** [review]

**Review by hrls — APPROVED** [review]

**Review by ales-tsurko — APPROVED** [review]

**Pull request merged**